spec/spec_helper_acceptance_litmus.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+require 'lib/dsc_utils_litmus'
+require 'erb'
+require 'serverspec'
+require 'puppet_litmus'
+require 'open3'
+include PuppetLitmus
+
+def run_local_command(command)
+  stdout, stderr, status = Open3.capture3(command)
+  error_message = "Attempted to run\ncommand:'#{command}'\nstdout:#{stdout}\nstderr:#{stderr}"
+  raise error_message unless status.to_i.zero?
+
+  stdout
+end
+
+def single_dsc_resource_manifest(dsc_type, dsc_props)
+  output = "dsc_#{dsc_type} {'#{dsc_type}_test':\n"
+  dsc_props.each do |k, v|
+    if v =~ /^\[.*\]$/
+      output += "  #{k} => #{v},\n"
+    elsif v =~ /^(true|false)$/
+      output += "  #{k} => #{v},\n"
+    elsif v =~ /^{.*}$/
+      output += "  #{k} => #{v},\n"
+    else
+      output += "  #{k} => '#{v}',\n"
+    end
+  end
+  output += "}\n"
+  return output
+end
+
+def test_file_path_manifest (test_dir_path, test_file_path, test_file_contents)
+  <<-Manifest
+  dsc_file {'tmp_folder':
+      dsc_ensure          => 'present',
+      dsc_type            => 'Directory',
+      dsc_destinationpath => "#{test_dir_path}",
+  }
+
+  dsc_file {'tmp_file':
+      dsc_ensure          => 'present',
+      dsc_type            => 'File',
+      dsc_destinationpath => "#{test_file_path}",
+      dsc_contents        => "#{test_file_contents}"
+  }
+  Manifest
+end
+
+def escape_value(value)
+  if is_number?(value)
+    value
+  else
+    "'#{value.to_s.gsub("'","''")}\'"
+  end
+end
+
+def is_number?(value)
+  value.respond_to?(:to_i) ? value.to_s == value.to_i.to_s : false
+end
+
+def create_remote_windows_file(folder_path, file_name, content)
```

Comment:
This might be worth moving to Litmus as a helper method to upload files so we don't have to keep repeating ourselves, I stole the code to call bolt form Litmus here: https://github.com/puppetlabs/puppet_litmus/blob/master/lib/puppet_litmus/rake_tasks.rb#L180

Reply:
these helper functions would be better suited moved out into a helper file.  ala https://github.com/puppetlabs/puppet_litmus/wiki/converting-tests-from-beaker-rspec-to-litmus#where-to-put-your-helper-functions have a look at https://github.com/puppetlabs/puppetlabs-postgresql/blob/master/spec/spec_helper_acceptance_local.rb

Reply:
Are there plans to move these helper functions to spec_helper_acceptance_local.rb @ThoughtCrhyme?

Reply:
Will do that in subsequent commits when we remove the beaker tests.